Documentation/Upgrade/ceph-upgrade.md

```diff
@@ -63,7 +63,7 @@ Official Ceph container images can be found on [Quay](https://quay.io/repository
 
 These images are tagged in a few ways:
 
-* The most explicit form of tags are full-ceph-version-and-build tags (e.g., `v17.2.6-20230410`).
+* The most explicit form of tags are full-ceph-version-and-build tags (e.g., `v17.2.7-20231122`).
```

Comment:
Let's not default to v17.2.7 due to #13136. How about this?
```suggestion
* The most explicit form of tags are full-ceph-version-and-build tags (e.g., `v17.2.6-20231027`).
```

Reply:
Good catch, yes!